### ISSUE TITLE
Incorporate help on creating an ACR

### ISSUE THREAD

**Issue by mgifford** [GSA/openacr#233]
From https://sewp.nasa.gov/documents/Section_508_Guide_111821.pdf

**What is an ACR?**
An Accessibility Conformance Report (ACR) is a document that provides information regarding the level of 
accessibility of a product based on the Section 508 Technical Standards. ~~This report is developed by Industry.~~

**HOW TO COMPLETE AN ACR**

The ACR is a representation of how the product meets the applicable Section 508 Technical Standards. The product 
owner/developer must first test the product against those standards in order to complete an ACR. This part of the guide will walk you through how to complete an ACR using OpenACR.

### **Step 1: Go to the OpenACR Editor**
The OpenACR is a free format with a free editor that helps you complete an ACR. Filling out the form in the editor will allow you to produce a validated OpenACR document, as well as an HTML version.

### **Step 2: Complete the OpenACR About Page**
The OpenACR process begins with an overview page that includes information and links several pages of instructions. On the About page you will need to provide information like:
• Company name
• Name of your product (and version number, if applicable)
• Report date (month and year)
• Description of the product
• Contact information
• Additional notes (if any)
• Evaluation methods: Information about how your product was tested (manual, automated, both), testing tools used, etc.

### **Step 3: Understanding  levels of conformance & comments for OpenACR tables**
 
For the category or categories which encompass the product, you must use one of four phrases—supports, partially 
supports, does not support, not applicable—to describe the level of conformance. 
• Supports: The functionality of the product has at least one method that meets the criterion without
known defects or meets with equivalent facilitation.
• Partially Supports: Some functionality of the product does not meet the criterion.
• Does Not Support: The majority of product functionality does not meet the criterion.
• Not Applicable: The criterion is not relevant to the product.

Note: The “Not Evaluated” phrase may only be used for the Level AAA table since this is the only success
criteria table that is not required to be completed. 

The third column is reserved for remarks, **which are required if the product either partially supports or does 
not support the guideline**. If the product supports a particular guideline, then no remarks are required but 
are encouraged. 

### **Step 4: Complete Table 1 (A) and Table 2 (AA) if applicable **
If your product fits into one or more of the following categories, then the WCAG 2.0 Standards apply to your 
product. 
1. Web Content
2. Electronic Documents
3. Software
4. Authoring Tool
If your product does not fall under any of these four categories, you may write “Not Applicable” in the notes 
section for the Success Criteria tables. 
The Success Criteria tables—Level A, Level AA, and Level AAA—cover the WCAG success criteria. 
Only Level A and AA are required in an ACR. _However, if your product does satisfy some (or all) Level AAA 
success criteria guidelines, then it is beneficial to complete that table as well, as it would provide additional 
motivation for Federal agencies to purchase your product and attract a wider range of potential customers.

### **Step 5: Determine the applicability of the Revised Section 508 Technical Standards to your product** 
There are four Revised Section 508 tables, labeled as Chapter 3 (Functional Performance Criteria), Chapter 4 
(Hardware), Chapter 5 (Software), and Chapter 6 (Support Documentation and Services). Complete the 
applicable tables.

If there are functions in your product not addressed by Chapters 4 or 5, then your product needs to conform 
to the criteria in Chapter 3: Functional Performance Criteria.
**Does your product contain hardware?**
Complete Chapter 4: Hardware. 
**Does your product contain non-web software?** 
Complete Chapter 5: Software. 
**Does your product contain support documentation and services?** 
Complete Chapter 6: Support Documentation and Services. 
**If your product cannot be evaluated using the current criteria,** 
Complete Chapter 3: Functional Performance Criteria. 
For each table, there is a section for notes. Use this field to enter comments about overall applicability of the 
table to your product.
The first column "Criteria" of the Revised Section 508 tables lists the Section 508 Technical Standards. By 
clicking on the links provided in this column, you may access more information about the standards on the 
U.S. Access Board website.

### **Step 6: Final checklist to make sure your OpenACR is complete** 
☐ Are you using the correct version? Make sure you are using the the latest version of OpenACR to build your ACR! If you have an older OpenACR file (ending with a .yaml extension) you should be able to **load it into the editor** to verify that it still validates. 
☐ Did you complete the information fields in the About tab of the editor? **You will see a warning if elements are not entered on this screen.** 
☐ If applicable, did you complete the Success Criteria, Level A and Level AA tables? If your product contains
web-based content, electronic documents, software, or authoring tools, make sure you complete the Level A
and Level AA tables.
☐ If applicable, did you complete the Chapter 3: Functional Performance Criteria table? If your product does
not contain any applicable standards from the Level A and AA tables, then complete the Chapter 3: Functional
Performance Criteria table.
☐ Did you skip the Level AAA tables? Section 508 does not require that your product be tested against the
Level AAA Technical Standards. _However, Industry may provide this information to show a higher level of
accessibility than required._
☐ If applicable, did you complete the tables under the Revised Section 508 Report section? If your product
contains hardware, software, or support documentation and services, be sure to complete the applicable
Revised Section 508 table.
☐ Are you using the correct phrases to describe conformance levels? Check to see if all phrases under
conformance level match one of the following: Supports, Partially Supports, Does Not Support, Not Applicable.
☐ Did you complete the remarks column? If the conformance level is either “Partially Supports” or “Does Not
Support,” then be sure that you explain further in the respective remarks column.

**Comment by danid123** [comment]
For 'Step 1: Go to the OpenACR Editor'
Can you change the text to:
The OpenACR is a free format with a free editor that helps you complete an ACR. Filling out the form in the editor will allow you to produce a validated OpenACR document, as well as an HTML version.

Also, are there any other benefits to using OpenACR we should highlight here?

**Comment by mgifford** [comment]
I changed that. Thanks. 

As far as other benefits, maybe. 

One of the benefits is that if content is being shipped back/fort between a vendor and whoever does their accessibility review (Deque, Level Access, etc), then it is easier for them to review & edit while retaining the format.  

Might be something that fits in the FAQ.

**Comment by mgifford** [comment]
Thanks for the wording changes. Always feel free to do that with my work. :)

Before posting it, might make sense to run. it through the HemingwayApp to make sure we are being more clear. 

I agree that having something might be useful. Maybe even just a page that has a few headings for the personas that we're thinking might use it. 

Are you a:
 - vendor's salesperson who has been asked to submit an OpenACR document?
 - vendor's accessibility subject matter expert (SME) who maintains your ACRs (including VPATs)?
 - accessibility SME on contract to a vendor
 - accessibility SME working for or on contract to an organization looking to purchase inclusive products and services? 
 
 Something like that that lest them know where to begin?

**Comment by danid123** [comment]
Add "company's" to this option: vendor's accessibility subject matter expert (SME) who maintains your company's ACRs (including VPATs)?

Question: for something like Drupal.org, or any open source organization where would they fall in this list of options?

**Comment by mgifford** [comment]
Thanks @danid123 I added that to the line above so it isn't missed. I similarly missed 

With open source projects, I think there might need to be some distinct recommendations.  Open source communities can leverage their issue queues. Also possible to simply point to open documentation that describes their processes.  Might also be recommendations on shaping issue tagging so that GitHub & GitLab projects can more easily track issues. 

Ultimately, it would be great if Drupal were able to demo this. Should be possible, but it will also be a lot of work.

**Comment by mgifford** [comment]
Brought over to the site in https://github.com/GSA/openacr-website/pull/41
